lighthouse-core/driver/gatherers/interstitial.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const Gather = require('./gather');
+
+/* global window, document, __returnResults */
+
+/* istanbul ignore next */
+function getInterstitial() {
+  const viewportSize = window.outerWidth * window.outerHeight;
+
+  const hasNonTransparentBackground = computedStyle => {
+    // Simple early exit for a solid background color.
+    const bgColor = computedStyle.backgroundColor;
+    const hasSolidBackground = (/#\d{6}/.test(bgColor) ||
```

Comment:
Most of this big section could be replaced with `return WebInspector.Color.parse(bgColor)._rgba[3] == 1`

However `WebInspector.Color` is available in LH and not in this clientside context. :/